index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>José De La Rosa</title>
+</head>
+<body>
+
+    <header>
+        <div class="logo"><a id="logo" href="#logo">My Logo</a></div>
+        <nav>
+            <button type="button" class="hamburger" id="openBtn"><img src="img/hamburger.png" alt="Menu icon"></button> 
+        </nav>
+    </header> 
+    
+    <section class="headline">
+        <div class="healine-content">
+            <h1>Hey there,
+                I’m Jose.
+                 </h1>
+            <h2>I’m a software developer!</h2>> 
+            <p>I can help you build a product, feature 
+                or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.
+            </p>
+        </div>
+
+        <ul class="social-links">
+            <li><a href="https://github.com/jrraul4" rel="noopener"><img src="img/github-icon.png" alt="Github icon" class="icon"></a></li>
```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design. Make sure that you adjust the color of the paragraph `I can help you ...`, fix the GitHub icon which is not showing, and the color of your social media icons to look exactly like the Figma design. :pray: 

| Your Design |Figma Design |
| :---:   | :---: | 
| ![image](https://github.com/jrraul4/Portafolio-Personal/assets/92303891/c4dfd251-c388-487f-bb2f-b28205115712)| ![image](https://github.com/jrraul4/Portafolio-Personal/assets/92303891/9df915d3-adc2-4938-8e4d-fa2e0a41715f)|

Reply:
@Owusu-Desmond Hi thank you for your review! I want to comment on your request:
1- in my pc the HTML open without problems and I can see the GitHub icon
2- in the paragraph "I can help you" I'm using the same color of the template which is #FFFFFF
3- I change the media icons with the right ones

waiting for your reply so I can submit again the project